### PR TITLE
add night menu

### PR DESCRIPTION
**Pull request by Chayapol-c**

### CODE REVIEW

File: Menu.md [hunk 1]
```diff
@@ -8,10 +8,21 @@
 
 ## Lunch Menu
 
+
+## Night Food Menu
+
+|         Food         |  Cost |
+|:--------------------:|-------|
+| Double Cheese Burger | 160   |
```
Comment (kidstylex) [inline Menu.md:16]: missing currency unit (THB)

File: Menu.md [hunk 2]
```diff
@@ -8,10 +25,18 @@
 
 ## Lunch Menu
 
-Coming soon.
 
-etc.
+## Night Food Menu
+
+|         Food         |  Cost  |
```
Comment (KOngTNP) [inline Menu.md:31]: change it to "| Price |"

File: Menu.md (hunk 1 above)
Comment (Chayapol-c) [reply Menu.md]: added already.

File: Menu.md (hunk 2 above)
Comment (Chayapol-c) [reply Menu.md]: Thank you. Ok, I've already edited.

**Comment by theethaj** [comment]
It's okay.

**Pull request merged**